_modules/win_pulsar.py

```diff
@@ -158,9 +158,10 @@ def process(configfile='salt://hubblestack_pulsar/hubblestack_pulsar_win_config.
     # Validate ACLs on watched folders/files and add if needed
     if update_acls:
         for path in config:
-            if path == 'win_notify_interval' or path == 'return' or path == 'batch' or path == 'checksum' or path == 'stats':
+            if path in ['win_notify_interval','return','batch','checksum','stats','paths','verbose']:
```

Comment:
Can you put a space after each comma? Just my pylint OCD showing up. Eventually I'd like to enable pylint on the repo and it will be one fewer thing to fix. :)

Reply:
pylint FTW!